examples/experimental/http-redirect-rewrite/httproute-redirect-prefix.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: gateway.networking.k8s.io/v1alpha2
```

Comment:
Mind adding backrefs like this here? https://github.com/kubernetes-sigs/gateway-api/blob/20683afe4018a42280d933971d44c258c328c964/examples/v1alpha2/basic-http.yaml#L1-L2